roles/plugins/foreman_openscap/tasks/main.yml

```diff
@@ -28,21 +28,54 @@
     {{ foreman_openscap_hammer }} proxy import-classes --name {{ foreman_openscap_proxy_name }}
   tags:
     - seed
-  when: '{{ foreman_openscap_proxy_name != False }}'
 
 - name: 'Import default scap content'
   shell: >
     foreman-rake foreman_openscap:bulk_upload:default
   tags:
     - seed
-  when: '{{ foreman_openscap_proxy_name != False }}'
-
-# TODO: disabled until cli is fixed
-# - name: 'Create example policy'
-#   shell: >
-#     {{ foreman_openscap_hammer }} policy create --name 'Example policy'
-#      --period weekly --weekday monday
-#      --scap-content 'Red Hat centos6 default content'
-#      --scap-content-profile-id TODO
-#   tags:
-#     - seed
+
+# Get the smart proxy ID of the local katello:
+- name: 'get scap content profile json'
+  shell: >
+    {{ foreman_openscap_hammer }} --output json scap-content info --title 'Red Hat centos7 default content'
+  register: foreman_openscap_scap_content_json
+
+- name: 'set openscap scap content'
+  set_fact:
+    foreman_openscap_scap_content: "{{ foreman_openscap_scap_content_json.stdout|from_json }}"
+
+- name: 'find CentOS policy'
+  shell: >
+    {{ foreman_openscap_hammer }} policy info --name 'Standard CentOS 7 OpenSCAP policy'
+  register: foreman_openscap_centos_policy_output
+  ignore_errors: True
+
+- name: 'create CentOS policy'
+  shell: >
+    {{ foreman_openscap_hammer }} policy create --name 'Standard CentOS 7 OpenSCAP policy' --period weekly --weekday monday --scap-content-id {{ foreman_openscap_scap_content.Id }} --scap-content-profile-id {{ foreman_openscap_scap_content | json_query('"SCAP content profiles"."1"."Id"') }}
```

Comment:
Will break when next hammer_cli_foreman_openscap is released, because fix for showing profiles as array is already in [master](https://github.com/theforeman/hammer_cli_foreman_openscap/commit/0740b3a77588fc94b0a83354edf76b77ecbcdc00)

Reply:
thanks, added a comment